lib/opensusebasetest.pm

```diff
@@ -1229,8 +1229,9 @@ sub remount_tmp_if_ro {
 
  select_serial_terminal($root);
 
-Select most suitable text console with root user. The choice is made by
-BACKEND and other variables.
+Select most suitable text console. By default the C<root> is used. To switch to a
+no-root user (usually this is C<testapi::username>, the C<select_serial_terminal> can
+accept a scalar 0 value. The choice is made by BACKEND and other variables.
```

Comment:
What is `a scalar 0 value`?

Reply:
an int 0

Reply:
I still don't know what this tells me here.

At a guess you could be meaning this: `C<select_serial_terminal> accepts a number corresponding to the user ID`

Reply:
What i try to document here is that `select_serial_terminal()` by default will give you a root terminal invoking the `root-virtio-terminal`. if it is called as `select_serial_terminal(0)` (note that 0 in perl is a scalar type) this shifts the selection to `user-virtio-terminal` which will give you a user's console. 
Let me try to submit a slightly different phrasing and we continue there.

Reply:
*This change is ready for review*

Reply:
please resolve and continue to https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/838b65095d71b4eea67010e0aac2a09a66ce9c40